Sources/PointFreeRouter/Routes.swift

```diff
@@ -190,7 +190,13 @@ private let routers: [Router<Route>] = [
     <% end,
 
   .webhooks <<< .stripe <<< .fallthrough
-    <¢> post %> lit("webhooks") %> lit("stripe") <% end,
+    <¢> post %> lit("webhooks") %> lit("stripe")
+    %> jsonBody(
```

Comment:
in a follow up let's also make this optional in case stripe is just sending bad json data